app/src/main/java/com/wonjoong/android/sopthub/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,34 @@
+package com.wonjoong.android.sopthub.ui.home
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import androidx.lifecycle.ViewModelProvider
+import com.wonjoong.android.sopthub.R
+import com.wonjoong.android.sopthub.databinding.ActivityHomeBinding
+import com.wonjoong.android.sopthub.util.BaseViewUtil
+
+class HomeActivity :
+    BaseViewUtil.BaseAppCompatActivity<ActivityHomeBinding>(R.layout.activity_home) {
+
+    private lateinit var viewModel: HomeViewModel
```

Comment:
선생님 activity-ktx와 fragment-ktx를 추가하셔서
```
dependencies {
    implementation 'androidx.activity:activity-ktx:버전은 몇 버전이 최신인지 모름'
    implementation 'androidx.fragment:fragment-ktx:버전은 몇 버전이 최신인지 모름'
}
```
이렇게 간결하게 표현도 가능합니다~!!
```suggestion
    private val homeViewModel : HomeViewModel by viewModels()
```